webpack.config.js

```diff
@@ -39,6 +45,24 @@ module.exports = env => {
         {
           test: /\.html?$/,
           use: [{ loader: 'html-loader' }]
+        },
+        {
+          test: /\.(css|scss|sass)$/i,
+          use: [
+            {
+              loader: 'style-loader'
+            },
+            {
+              loader: 'css-loader'
+            },
+            {
+              loader: 'sass-loader'
```

Comment:
I am thinking we will not need this.  This project will be using glamorous for styles that do not come from Patternfly:  https://glamorous.rocks/

Reply:
I need to check it again because yarn start throw errors about to add a loader to handle this type of files, I think that patternfly-react css import them

Reply:
css shouldn't need sass-loader.  We do need style-loader if we do not use the css plugin.  An example of the config for the css plugin would look like this:
```javascript
// webpack.config.js
module.exports = () => {
  return {
    ...rest of config,
    module: {
       rules: [
         ...rest of rules,
         {
            test: /\.css$/,
            use: [MiniCssExtractPlugin.loader, 'css-loader']
         },
       ]
    },
    plugins: [
        ...rest of plugins,
        new MiniCssExtractPlugin({
           filename: '[name].css',
           chunkFilename: '[id].css'
        })
    ],
  }
}

```